docs/fx-core/switch-account-or-subscription-help.md

```diff
@@ -0,0 +1,68 @@
+This doc is to help you understand what will happen when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365(M365) account. We will also explain how to recover from the backups in this doc.
+
+> Important Notes: After switching accounts and provisioning or local debugging again, resources have been created before in the old M365 tenant or Azure subscription won't be deleted by default, and you have to manully delete them to avoid further costs if any. 
+
+## Switch Microsoft 365 account
+### Local debug
+You could run local debugging for a Teams project with one M365 account and then easily switch to another account for further local debugging. To do this, you only need to:
+1. Sign out of the current M365 account.
+2. Sign in to the new account.
+3. Start local debugging.
+
+After that, we will 
+1. Back up configuration files for local environment. [Learn more about backup & recover](#backup--recover).
```

Comment:
Can we be more explicit on what exactly are the files that will be backed up? People have different understanding on `configuration files`.

Reply:
the detailed files are described in the link. [Learn more about backup & recover](#backup--recover).